server/models/pages.js

```diff
@@ -926,6 +927,29 @@ module.exports = class Page extends Model {
     return rebuildJob.finished
   }
 
+  /**
+   *
+   * @returns
+   */
+  static async buildXMLSitemap() {
```

Comment:
Should be part of the controller directly as it's a single purpose method.

Reply:
Gotcha!